### DISCUSSION TITLE
How to consume style dictionary tokens in app

### DISCUSSION THREAD

**Discussion by ChellappanRajan** [nxkit/nxkit#92] [Q&A]
How do I use/style dictionary tokens generated using the NXKit plugin in my nx apps? Can you please provide guidance?

**Comment by ChellappanRajan** [comment]
@JoepKockelkorn @sebastiandg7 Can you please share any tips or sample repo I could look into?

**Reply by JoepKockelkorn** [reply]
I don't know anything about that plugin so I can't help you.
